crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -644,19 +645,19 @@ impl<B: BlockT> futures::Future for SubspaceWorker<B> {
     }
 }
 
-struct SubspaceSlotWorker<B: BlockT, C, E, I, SO, L, BS> {
-    client: Arc<C>,
-    block_import: I,
-    env: E,
-    sync_oracle: SO,
-    justification_sync_link: L,
-    force_authoring: bool,
-    backoff_authoring_blocks: Option<BS>,
-    subspace_link: SubspaceLink<B>,
-    signing_context: SigningContext,
-    block_proposal_slot_portion: SlotProportion,
-    max_block_proposal_slot_portion: Option<SlotProportion>,
-    telemetry: Option<TelemetryHandle>,
+pub(crate) struct SubspaceSlotWorker<B: BlockT, C, E, I, SO, L, BS> {
+    pub(crate) client: Arc<C>,
+    pub(crate) block_import: I,
+    pub(crate) env: E,
+    pub(crate) sync_oracle: SO,
+    pub(crate) justification_sync_link: L,
+    pub(crate) force_authoring: bool,
+    pub(crate) backoff_authoring_blocks: Option<BS>,
+    pub(crate) subspace_link: SubspaceLink<B>,
+    pub(crate) signing_context: SigningContext,
+    pub(crate) block_proposal_slot_portion: SlotProportion,
+    pub(crate) max_block_proposal_slot_portion: Option<SlotProportion>,
+    pub(crate) telemetry: Option<TelemetryHandle>,
```

Comment:
I do not think these actually need to be `pub(crate)`, it should be visible without to any nested module.